sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -823,6 +824,27 @@ def only_element(iterable):
   return element
 
 
+def _verify_descriptor_created_in_a_compatible_env(process_bundle_descriptor):
+  # type: (beam_fn_api_pb2.ProcessBundleDescriptor) -> None
+
+  runtime_sdk = environments.sdk_base_version_capability()
+  for _, t in process_bundle_descriptor.transforms.items():
```

Comment:
You can also do `for t in process_bundle_descriptor.transforms.values()`

Reply:
fixed, thanks.